test/analysis/SmartDetectorsAnalysisTests/AlertPresentationTests.cs

```diff
@@ -276,5 +305,34 @@ public class TableData
 
             public string NonDisplayProp { get; set; }
         }
+
+        [SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic", Justification = "Test code, approved")]
+        public class ListData1
+        {
+            [TextProperty("Name1", Order = 1)]
+            public string Name1 => "Name1";
+
+            [UrlFormatter("Link to data 1")]
+            [TextProperty("Link1", Order = 2)]
+            public Uri Uri1 => new Uri("https://xkcd.com/");
+        }
+
+        [SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic", Justification = "Test code, approved")]
+        public class ListData2
+        {
+            [TextProperty("Name2", Order = 1)]
+            public string Name2 => "Name2";
```

Comment:
>Name2 [](start = 26, length = 5)

what if this was named Name1?
i think we should fully qualify the property name